C-Plus-Plus/dp/SlidingWindowTechnique.cpp

```diff
@@ -0,0 +1,44 @@
+#include <algorithm>
```

Comment:
You can just use *#include <bits/stdc++.h>* instead of adding header files individually.

Reply:
And add description of sliding window technique at the top with its time and space complexity.